raiden/utils/serialization.py

```diff
@@ -4,28 +4,33 @@
 from eth_utils import to_bytes, to_canonical_address, to_checksum_address, to_hex
 
 from raiden.transfer.merkle_tree import LEAVES, compute_layers
-from raiden.utils import typing
+from raiden.utils.typing import Address, Callable, Dict, List, Tuple, TypeVar
 
+T = TypeVar('T')
```

Comment:
The names stands for:

`T` for `type`
`KT` for `key type`
`VT` for `value type`
`RT` for `return type`

Reply:
This is a bit complicated. At the least you should add your github comment as a comment in the code itself so that others can understand the reasoning behind the naming. Shouldn't this also be moved into the generic typing module?

I also wouldn't mind just naming them `X`,`Y`,`Z`, `W` and then just using them like that, it's easy to understand what they stand for once you read the function and see that different generics are used for different arguments.

But no strong opinions. I am fine with it as long as there is a comment explaining what this is.

Reply:
https://mypy.readthedocs.io/en/latest/generics.html#defining-sub-classes-of-generic-classes `T`, `KV`, `VT` are commonly used variable names, I did not chose them